src/main/java/com/microfocus/application/automation/tools/run/PcBuilder.java

```diff
@@ -262,13 +262,25 @@ private Testsuites run(PcClient pcClient, Run<?, ?> build)
         boolean trendReportReady = false;
         try {
             runId = pcClient.startRun();
-            testName = pcClient.getTestName();
+            try {
+                testName = pcClient.getTestName();
+                logger.println(String.format("%s - test name is %s \n", _simpleDateFormat.format(new Date()), testName));
+            }
+            catch (Exception ex) {
```

Comment:
getTestName throws IOException and PCException

Reply:
indeed. I just don't want to have the job failing because we failed to get the name of the test.

Reply:
So there is no need to catch all types?